<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCloudProviders</class>
 <widget class="QDialog" name="NewCloudProviders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}
QCommandLinkButton
{
}
QDialog
{
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

}

QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #28d, stop: 0.1 #39e, stop: 0.49 #47c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;
min-width: 70px;
max-width:70px;
min-height: 17px;
max-height: 17px;
}</string>
  </property>
  <widget class="QLabel" name="providerNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Provider Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="providerURLLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Provider URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="providerTypeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Provider Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="xDeltaCloudProviderLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Deltacloud-Provider</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cloudProviderTypeComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="providerNameLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="providerURLLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xDeltaCloudProviderLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
